packages/app-store/templates/audit-log-implementation/config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This is a template, will be loaded with all that is needed for a third party integration.